fontc/src/workload.rs

```diff
@@ -531,20 +548,33 @@ impl<'a> Workload<'a> {
                                 }
                             }
                             let timing = timing.complete();
+
                             if let Err(e) = send.send((id.clone(), result, timing)) {
                                 log::error!("Unable to write {id:?} to completion channel: {e}");
                             }
                         })
                     }
+                    self.timer.add(timing.complete());
                 }
 
-                // Block for things to phone home to say they are done
-                // Then complete everything that has reported since our last check
+                // Complete everything that has reported since our last check
                 if successes.is_empty() {
+                    let timing = create_timer(AnyWorkId::InternalTiming("rc"), nth_wave)
+                        .queued()
+                        .run();
```

Comment:
I think tracing is going to become very important for us and we should investigate current good practices. back in the xi days we wrote [`xi_trace`](https://docs.rs/xi-trace/latest/xi_trace/) which was very useful, but there are probably better alternatives now.

Reply:
Flamegraph is awesome but at times over-broad. What I'd really like right now is the ability to drill from `--emit-timing` => flamegraph of just that task.

Reply:
https://github.com/googlefonts/fontc/issues/646